docs/docs-content/audit-logs/monitor-with-datadog.md

```diff
@@ -0,0 +1,167 @@
+    | -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | **pack.namespace**                           | Provide a namespace for the Datadog installation, for example, `datadog`.                                                                                                                                         |
+    | **datadog.apiKey**                           | Insert your Datadog API key.                                                                                                                                                                                      |
+    | **datadog.site**                             | Insert your regional Datadog website, for example, `us5.datadoghq.com`. Refer to the [Getting Started with Datadog Sites](https://docs.datadoghq.com/getting_started/site/) page for a complete list of websites. |
+    | **datadog.logs.enabled**                     | Set it to `true` to enable the Datadog Agent log collection.                                                                                                                                                      |
+    | **datadog.logs.containerCollectAll**         | Set it to `true` to allow log collection for all containers.                                                                                                                                                      |
+    | **datadog.processAgent.processCollection**   | Set it to `true` to enable process collection.                                                                                                                                                                    |
+    | **datadog.networkMonitoring.enabled**        | Set it to `true` to enable network performance monitoring.                                                                                                                                                        |
+    | **datadog.serviceMonitoring.enabled**        | Set it to `true` to enable universal service monitoring.                                                                                                                                                          |
+    | **datadog.securityAgent.compliance.enabled** | Set it to `true` to enable the security agent.                                                                                                                                                                    |
+
+    Paste the following snippet under the `clusterAgent.volumes` section to enable the volume and volume mount for
```

Comment:
There is a lot going on in step 11. My advice is to break it up into three separate steps. Keep the table, the next step can be the volume path, and the third step can be the namespace label guidance.

Reply:
Oh yes. Done  :white_check_mark: